Jetcaster/app/src/main/java/com/example/jetcaster/util/WindowInfoUtil.kt

```diff
@@ -31,7 +31,14 @@ sealed interface DevicePosture {
         val hingePosition: Rect
     ) : DevicePosture
 
-    object BookPosture : DevicePosture
+    data class BookPosture(
+        val hingePosition: Rect
+    ) : DevicePosture
+
+    data class Separating(
```

Comment:
Nit: Rename `SeparatingPosture`? To keep it consistent with the other names

Reply:
Ack